automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/FreeTrialSpec.scala

```diff
@@ -0,0 +1,105 @@
+package org.broadinstitute.dsde.firecloud.test.user
```

Comment:
#1519 deleted this entire file; I'm now putting it back,  but have refactored it. Might be easiest to compare this to the commit prior to  #1519: https://github.com/broadinstitute/firecloud-ui/compare/a4d39cc0f197a70b87c1b731891722ab73473ca2...da_trialBannersBack